### PR TITLE
break:  rename functions in UserMenu props

### PR DESCRIPTION
**Pull request by braianj**

### CODE REVIEW

File: src/components/UserMenu/UserMenu.types.ts [hunk 1]
```diff
@@ -19,26 +19,26 @@ export type UserMenuProps = Omit<
 
 export type UserMenuI18N = Record<
   | 'myAssets'
-  | 'settings'
+  | 'account'
```
Comment (AlejandroAlvarezMelucciDCL) [inline src/components/UserMenu/UserMenu.types.ts:22]: Are you sure you want to remove `settings` and duplicate `account`?

Comment (braianj) [reply src/components/UserMenu/UserMenu.types.ts:22]: yeah, and thanks for the duplication catch.

**Comment by AlejandroAlvarezMelucciDCL** [comment]
LGTM but I won't approve since I lack enough knowledge of this repository and the systems involved, let's wait for someone with more experience 💪

File: src/components/UserMenu/UserMenu.types.ts [hunk 2]
```diff
@@ -19,26 +19,25 @@ export type UserMenuProps = Omit<
...
   ACTIVITY = 'activity',
+  ACCOUNT = 'account',
   MY_ASSETS = 'my_assets',
-  SETTINGS = 'settings',
   PROFILE = 'profile',
   SIGN_IN = 'sign_in',
   SIGN_OUT = 'sign_out',
   GUEST = 'guest',
-  WALLET = 'wallet',
+  MARKETPLACE_AUTHORIZATION = 'marketplace_authorization',
```
Comment (LautaroPetaccio) [inline src/components/UserMenu/UserMenu.types.ts:40]: Shouldn't this be in its plural form?
```suggestion
  MARKETPLACE_AUTHORIZATIONS = 'marketplace_authorizations',
```

**Review by LautaroPetaccio — COMMENTED** [review]
Looks great! I've left just a comment to be reviewed

**Review by LautaroPetaccio — APPROVED** [review]

**Pull request merged**